samples/golf/CMakeLists.txt

```diff
@@ -54,7 +52,7 @@ find_package(OpenGL REQUIRED)
 find_package(Bullet REQUIRED)
 find_package(SQLite3 REQUIRED)
 
-if(USE_NEWSFEED)
+if(NOT USE_STEAM)
```

Comment:
Will check but at first glance not sure this is correct - I know the RSS/GNS stuff used to be mutually exclusive but now they can both be enabled so think this check just needs removing so we always try to find curl?

Reply:
Oh, I meant to include this in my comment, but I guess I forgot - Steam uses its own version of curl internally, so we can't link curl in Steam builds, it causes a conflict.

Reply:
Ah I see - makes sense, guess I was getting away with it more by luck than skill.

Otherwise all still seems fine with your changes